src/projectM-sdl/pmSDL.hpp

```diff
@@ -88,6 +88,8 @@ class projectMSDL : public projectM {
 
 
     bool done;
+    bool wasapi = false; // Used to track if wasapi is currently active. This bool will allow us to run a WASAPI app and still toggle to microphone inputs.
+    bool fake_audio = false; // Used to track fake audio, so we can turn it off and on.
```

Comment:
i guess it should be `fakeAudio` for consistency
```suggestion
    bool fakeAudio = false; // Used to track fake audio, so we can turn it off and on.
```